hoomd/hpmc/pytest/test_compute_sdf.py

```diff
@@ -14,7 +14,7 @@
 def test_before_attaching():
```

Comment:
Add a unit test to ensure that the linear search code path executes and computes the correct values.

Reply:
Done